packages/teams-js/src/public/dialog.ts

```diff
@@ -233,15 +233,15 @@ export namespace dialog {
    * @hidden
    * Hide from docs
    * --------
-   * Convert UrlDialogInfo to DilogInfo to send the information to host in {@linkcode open} API.
+   * Convert UrlDialogInfo to DialogInfo to send the information to host in {@linkcode open} API.
    *
    * @internal
    */
   export function getDialogInfoFromUrlDialogInfo(urlDialogInfo: UrlDialogInfo): DialogInfo {
     const dialogInfo: DialogInfo = {
       url: urlDialogInfo.url,
-      height: urlDialogInfo.size.height,
-      width: urlDialogInfo.size.width,
+      height: urlDialogInfo.size ? urlDialogInfo.size.height : DialogDimension.Small,
```

Comment:
This might be a stupid question, but how did we decide to default to Small if no size is provided? Is this matching legacy behavior?

Reply:
We just picked "small" as default in our meeting with partners.
length/width are optional in tasks but, the "size" is a required parameter in dialogs. That means there should always be something in height/width. 
because of that I thought we would not need to check for urlDialogInfo.size ?, but seems like things still somehow can make through without providing size. Hence this extra checking.